### PR TITLE
Refine UT and logs

### PR DESCRIPTION
**Pull request by fyrestone**
## What do these changes do?

- Some UTs need specific version of third party lib, e.g. ray.
- Some logs may be too long.

## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://docs.pymars.org/en/latest/development/contributing.html#check-code-styles) for how to run them

### CODE REVIEW

**Review by zhongchun — APPROVED** [review]
LGTM.

File: mars/dataframe/datasource/tests/test_datasource_execution.py [hunk 1]
```diff
@@ -52,6 +52,10 @@
 
 ray = lazy_import("ray")
 _date_range_use_inclusive = pd_release_version[:2] >= (1, 4)
+try:
+    import ray.data as ray_dataset
+except Exception:
```
Comment (wjsi) [inline mars/dataframe/datasource/tests/test_datasource_execution.py:57]: Define error types.

File: mars/core/entity/chunks.py [hunk 2]
```diff
@@ -55,5 +58,11 @@ def _update_key(self):
 class Chunk(Entity):
     _allow_data_type_ = (ChunkData,)
 
+    def __str__(self):
+        return f"{type(self).__name__}({self._data.__str__()})"
+
+    def __repr__(self):
+        return f"{type(self).__name__}({self._data.__repr__()})"
```
Comment (wjsi) [inline mars/core/entity/chunks.py:65]: When `__ str__` is not defined, `__repr__` will be used. Hence no redundant methods needed.

File: mars/dataframe/datasource/tests/test_datasource_execution.py (hunk 1 above)
Comment (fyrestone) [reply mars/dataframe/datasource/tests/test_datasource_execution.py]: Nice catch!

**Review by chaokunyang — APPROVED** [review]
LGTM

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**